.changeset/wicked-ducks-wink.md

```diff
@@ -0,0 +1,17 @@
+---
+'@sl-design-system/editorial-suite': patch
+'@sl-design-system/my-digital-book': patch
+'@sl-design-system/sanoma-learning': patch
+'@sl-design-system/itslearning': patch
+'@sl-design-system/bingel-int': patch
+'@sl-design-system/bingel-dc': patch
+'@sl-design-system/clickedu': patch
+'@sl-design-system/magister': patch
+'@sl-design-system/kampus': patch
+'@sl-design-system/neon': patch
+'@sl-design-system/teas': patch
+'@sl-design-system/max': patch
+'@sl-design-system/tig': patch
+---
+
+Add missing package exports for CSS files
```

Comment:
Can you please also update Getting started page for developers on the documentation website:
https://sanomalearning.design/categories/getting-started/developers/ ?

I mean this section:
https://sanomalearning.design/categories/getting-started/developers/#setup-a-theme
that styling can be used also by importing explicitly those files (not only from node_modules as it is described right now).

And also this section:
https://sanomalearning.design/categories/getting-started/developers/#how-do-i-use-the-dark-mode-of-the-theme